### PR TITLE
Add test for unsigned long #978

### PR DESCRIPTION
**Pull request by vishnumohan1991**
Added a test for unsigned long in the below file under `tests/oeedger8r`.

```
tests/oeedger8r/host/testarray.cpp
tests/oeedger8r/enc/testarray.cpp
tests/oeedger8r/host/testpointer.cpp
tests/oeedger8r/enc/testpointer.cpp
```

Test for unsigned char, unsigned short & unsigned int covered under ticket #719

### CODE REVIEW

**Comment by vishnumohan1991** [comment]
bors try

File: tests/oeedger8r/host/testbasic.cpp [hunk 1]
```diff
@@ -151,16 +151,16 @@ void test_basic_edl_ecalls(oe_enclave_t* enclave)
 
     if (g_enabled[TYPE_LONG])
     {
-        long double ret = 0;
-        OE_TEST(ecall_ret_long_double(enclave, &ret) == OE_OK);
-        OE_TEST(ret == 0.191919);
+        long ret = 0;
+        OE_TEST(ecall_ret_long(enclave, &ret) == OE_OK);
```
Comment (anakrish) [inline tests/oeedger8r/host/testbasic.cpp]: Good catch.

**Comment by berinpaul** [comment]
bors try

**Comment by vishnumohan1991** [comment]
bors try

**Comment by vishnumohan1991** [comment]
bors try

**Pull request merged**